CODE_OF_CONDUCT.md

```diff
@@ -1,127 +0,0 @@
-# Contributor Covenant Code of Conduct
```

Comment:
Why this got deleted? Do any of Weblate's policies requires us to do so? Or is it just for brevity?

Reply:
I deleted it so I can contribute in good conscience.
Weblate is libre software. It neither has a CoC, nor enforces one as such.
In the interest of brevity, I think it does nothing productive vis-à-vis having a clear contact field option.

Reply:
> I deleted it so I can contribute in good conscience. Weblate is libre software. It neither has a CoC, nor enforces one as such. In all manners pertaining to brevity, I think a this CoC does nothing productive vis-à-vis a clear contact field.

Okay, makes sense. We defenitely can have CONTACT.md. But, I don't think there's a need to remove CODE_OF_CONDUCT.md for this. Can you please revert the deletion? Thanks

Reply:
I don't want any contributions of mine in a project that has this CoC.

Reply:
I think you are intending to say - we need only a pledge and a contact field and not any restrictions like bans as such on this OSS project. Right? Merging...